### DISCUSSION TITLE
Compose guidelines

### DISCUSSION THREAD

**Discussion by JavierSegoviaCordoba** [facebookincubator/ktfmt#191] [Ideas]
I think the format is following the Kotlinlang one at 100%, but I want to share it because maybe I am missing something.

https://github.com/androidx/androidx/blob/androidx-main/compose/docs/compose-api-guidelines.md

**Comment by hick209** [comment]
I am not sure about the goal of this discussion here, so I'll close it (also because it's old).
Feel free to reopen with more info / context if necessary.
